### ISSUE TITLE
hspec-contrib 0.5.1 tests:Tree compile error on ghc 9.4.2 / hspec-core 2.10.6

### ISSUE THREAD

**Issue by danwdart** [hspec/hspec#747]
Hello, don't know what's going on here, but looks like a different use of Tree?

```
test/Test/Hspec/Contrib/HUnitSpec.hs:11:68: error:
    • Couldn't match type: (base-4.17.0.0:Data.Semigroup.Internal.Endo
                              hspec-core-2.10.6:Test.Hspec.Core.Config.Definition.Config,
                            [SpecTree ()])
                     with: [Tree a0 (Item a1)]
      Expected: IO [Tree a0 (Item a1)]
        Actual: IO
                  (base-4.17.0.0:Data.Semigroup.Internal.Endo
                     hspec-core-2.10.6:Test.Hspec.Core.Config.Definition.Config,
                   [SpecTree ()])
    • In the second argument of ‘(<$>)’, namely
        ‘runSpecM (fromHUnitTest a)’
      In the first argument of ‘shouldReturn’, namely
        ‘map (bimapTree (const ()) itemRequirement)
           <$> runSpecM (fromHUnitTest a)’
      In the expression:
        map (bimapTree (const ()) itemRequirement)
          <$> runSpecM (fromHUnitTest a)
          `shouldReturn` b
   |
11 | a `shouldYield` b = map (bimapTree (const ()) itemRequirement) <$> runSpecM (fromHUnitTest a) `shouldReturn` b
```

Thanks for any help.

**Comment by sol** [comment]
Fixed by `hspec-contrib-0.5.1.1` and Hackage revisions.  Thanks for reporting his.

**Comment by danwdart** [comment]
Thanks, that was fast!
